src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1205,11 +1207,6 @@ internal static int GetBinIdx(float output, float min, float binSize, int numBin
             return binIdx;
```

Comment:
Ugh. This code makes my eyes bleed but if no one has complained after all these years we can probably let it stand for a bit. :)